core/src/avm2/globals/xml.rs

```diff
@@ -830,6 +830,42 @@ pub fn set_children<'gc>(
     Ok(xml.into())
 }
 
+// ECMA-357 13.4.4.34 XML.prototype.setLocalName ( name )
```

Comment:
formatting nit: `in other places I see this written as `XML.prototype.setLocalName (name)`

Reply:
Well, we have instances of both, for example: https://github.com/ruffle-rs/ruffle/blob/a638715bd87d818d134a70659e7124234b992447/core/src/avm2/globals/xml.rs#L489

Reply:
Okay then, never mind.